linkis-computation-governance/linkis-computation-governance-common/src/main/scala/org/apache/linkis/governance/common/utils/GovernanceUtils.scala

```diff
@@ -73,13 +73,41 @@ object GovernanceUtils extends Logging {
     }
   }
 
+  def killProcessByPort(port: String, desc: String, isSudo: Boolean): Unit = {
+    val subProcessKillScriptPath = Configuration.getLinkisHome() + "/sbin/kill-process-by-port.sh"
+    if (
+        StringUtils.isBlank(subProcessKillScriptPath) || !new File(subProcessKillScriptPath)
+          .exists()
+    ) {
+      logger.error(s"Failed to locate kill-script, $subProcessKillScriptPath not exist")
+    } else if (StringUtils.isNotBlank(port)) {
+      val cmd = if (isSudo) {
+        Array("sudo", "sh", subProcessKillScriptPath, String.valueOf(port))
```

Comment:
Input string type about  port   parameter
String.valueOf(port)   can be removed

Reply:
yes